src/index.html

```diff
@@ -21,7 +24,401 @@
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__product-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__product-price">
+          <p class="price-title">Price:</p>
+          <p class="price-numbers">$2,199</p>
+        </div>
+
+        <a
+          class="card__button-buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__product-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__product-price">
+          <p class="price-title">Price:</p>
+          <p class="price-numbers">$2,199</p>
+        </div>
+
+        <a
+          class="card__button-buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__product-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__product-price">
+          <p class="price-title">Price:</p>
+          <p class="price-numbers">$2,199</p>
+        </div>
+
+        <a
+          class="card__button-buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The product cards are repeated multiple times with the same content. Consider using a templating engine or JavaScript to dynamically generate these cards if the content is meant to be the same, or ensure that each card has unique content if they represent different products.